### ISSUE TITLE
Top rated posts over time period

### ISSUE THREAD

**Issue by Ramblurr** [hnrss/hnrss#54]
I would like to cut down my daily HN reading and instead consume a list of articles once per week. Sort of like a weekly HN magazine.

cperciva has had a weekly top list for some time http://www.daemonology.net/hn-weekly-show/

It's notable because it uses a fuzzy notion.of week, such that a post that is posted right before the deadline that later goes on to receive enough votea will still be featured.

What I want is the ability to configure the number of posts in this list. cperciva's is fixed to 10.

Is this possible with hnrss'? 

Something like a feed that is updated once per week. Or a feed like cperciva's that includes links to the items.

**Comment by joshua-tree-19231** [comment]
Agreed. Another use case would be a daily digest of top HN posts in the past 24h, like Morning Brew does. <img width="782" alt="Screenshot 2023-11-21 at 15 00 12" src="https://github.com/hnrss/hnrss/assets/1293565/97553541-f87b-4239-a38b-228d35c88f16">
